app/controllers/alchemy/pages_controller.rb

```diff
@@ -156,7 +156,7 @@ def legacy_urls
       # /slug/tree => slug/tree
       urlname = (request.fullpath[1..-1] if request.fullpath[0] == '/') || request.fullpath
 
-      LegacyPageUrl.joins(:page).where(urlname: urlname, alchemy_pages: {language_id: Language.current.id})
+      LegacyPageUrl.joins(:page).where(urlname: urlname, Page.table_name => {language_id: Language.current.id})
```

Comment:
Line is too long. [111/80]<br>Space inside { missing.<br>Space inside } missing.